go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/decred/vspd
 
-go 1.20
```

Comment:
Same.  No reason to bump this.

Reply:
copyloopvar requires it.

Reply:
It doesn't seem very prudent to me to require a new version of Go for all consumers just to add a linter.